hadoop-ozone/dist/src/main/smoketest/admincli/container.robot

```diff
@@ -96,6 +96,33 @@ List all containers from a particular container ID
     ${output} =         Execute          ozone admin container list --all --start 1
                         Should contain   ${output}   OPEN
 
+List containers in JSON array format
+    ${output} =         Execute          ozone admin container list --json | jq -r '.'
+                        Should Start With   ${output}   [
```

Comment:
This way of testing works, but a jq query to do the same test might be more robust (although failure messages may be more confusing):
`ozone admin container list --json | jq '.[] | .containerID'` then ensure that we actually get content in the output.

Reply:
The goal is to see if the parsing matches, other tests delve deeper into the returned payload. I can add some additional checks here as well.

Reply:
> `ozone admin container list --json | jq '.[] | .containerID'`

I suggest splitting it into two commands:

```
${output} =     Execute    <command returns JSON>
${excerpt} =    Execute    echo '${output}' | jq '....'
```

This has the benefit of showing raw output, which helps in case `jq` fails to find expected structure.  Plus you can reuse `output` for multiple extractions.